Automattic/jetpack
# AI Assistant: Add styles to forms

Fixes #32151

## Proposed changes:

* Tweaks the prompt to add styles

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Test the form as normal, there should be no regressions
* Ask for styles on new forms and on existing forms

To help testing: "Make a pizza ordering form with dark yellow background and red text", "make the borders rounded".
